### PR TITLE
Add watchOS app + other improvements

### PR DESCRIPTION
**Pull request by majouji**
- Add watchOS app!
- Exclude unsupported components with `#if os` statements
- Update Readme + add useful Apple links
- Add system images section in the Images group
- Refactor `var lineColor` to be a color set in the asset library instead (initially to avoid a big, ugly if else block but seems to be the better approach)
- Add `renderingMode(.original)` modifier example for multi-color system images
- Add frame constraints to the shapes examples so they don’t stretch out too much when resizing windows on macOS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # SwiftUI Kit
-A SwiftUI system components and interactions demo app based on iOS 14 and macOS Big Sur
+A SwiftUI system components and interactions demo app based on iOS 14, macOS Big Sur, and watchOS 4.
```
Comment (jordansinger) [inline README.md:2]: watchOS 7?

File: README.md [hunk 2]
```diff
@@ -19,9 +19,12 @@ To change the accent color and see how it affects the system components, go to t
 Please feel free to submit pull requests in order to contribute back to SwiftUI Kit. This is for the SwiftUI community!
 
 #### SwiftUI Resources
-[Recreate](https://recreatecode.com) - A video series about recreating popular UI with SwiftUI
+- [Recreate](https://recreatecode.com) - A video series about recreating popular UI with SwiftUI
+- [Primitive](https://primitive.school) - Learn SwiftUI for designers
 
-[Primitive](https://primitive.school) - Learn SwiftUI for designers
+#### Apple Resources
```
Comment (jordansinger) [inline README.md:25]: thanks for adding!

File: SwiftUI Kit/Groupings/ShapesGroup.swift [hunk 3]
```diff
@@ -8,13 +8,24 @@
 import SwiftUI
 
 struct ShapesGroup: View {
+    // Define constraints for the shape frames so they don’t stretch out too much on macOS
```
Comment (jordansinger) [inline SwiftUI Kit/Groupings/ShapesGroup.swift:11]: thanks for doing this!

File: SwiftUI Kit/Groupings/ImagesGroup.swift [hunk 4]
```diff
@@ -17,7 +17,20 @@ struct ImagesGroup: View {
                     Image("Image")
                         .resizable()
                         .aspectRatio(contentMode: .fit)
-                        .frame(height: 128)
+                        .frame(maxHeight: 128)
+                }
+            )
+            
+            SectionView(
+                title: "System Images",
```
Comment (jordansinger) [inline SwiftUI Kit/Groupings/ImagesGroup.swift:25]: awesome

**Comment by majouji** [comment]
(Don’t merge yet – verifying that stuff merged from master still works properly)

**Comment by majouji** [comment]
Good now, but I don’t have permission to merge!

**Pull request merged**